janus_core/Cargo.toml

```diff
@@ -58,6 +58,6 @@ hex = { version = "*", features = ["serde"] }
 janus_core = { path = ".", features = ["test-util"] }
 # Enable `kube`'s `openssl-tls` feature (which takes precedence over the
 # `rustls-tls` feature when creating a default client) to work around rustls's
-# lack of support for IP addresses in certificates when connecting to a Kind
-# cluster.
+# lack of support for IP addresses in certificates when connecting to most
```

Comment:
I think the specific condition is if the cluser is identified by IP, rather than DNS name -- IIUC rustls does not yet support IP-address-based certs.

Maybe "Kubernetes clusters identified by IP" or "local Kubernetes clusters" instead of "most Kubernetes clusters"? Both are more specific to the conditions that cause breakage.